frontend/src/icons/userLogo/UserLogo.tsx

```diff
@@ -0,0 +1,56 @@
+//Libs
+import React, { useState, useMemo } from 'react';
+
+//Type
+import { User } from '../../components/body/types/user';
```

Comment:
??

Reply:
extendible banane ke liye, abhi static use image and username hai, later, we would want to use it like 
<UserLogo user={particularUser} />

Reply:
types should be uplifted